src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -565,7 +565,7 @@ static int add_partition_xbootldr(DissectedPartition *p) {
 static int add_partition_esp(DissectedPartition *p, bool has_xbootldr) {
```

Comment:
commit message: `an fstab entry` -> `a fstab entry`

Reply:
No? `f` starts with a vowel: https://dictionary.cambridge.org/pronunciation/english/f

Reply:
haha, I might be pronounced that [fistæb] instead of [efestæb] when I reviewed :-p
Sorry for the noise.